pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -683,6 +683,7 @@ def __init__(
 
         self.nevra_to_module = defaultdict(dict)
         self.pkgname_to_groups = defaultdict(list)
+        self.forbidden_metadata_types = ["prestodelta", "deltainfo"]
```

Comment:
This is specific to the `mirror_complete` mode so I'm not sure it makes sense to be a class-wide variable.  I would just inline the list into the check location.  We don't expect it to get longer than this.

Reply:
Or rather: since it's used in more than one place, it can be a global constant named `UNSUPPORTED_METADATA` or something.

Reply:
Or a class global.  In any case making it an instance variable communicates the wrong message I think.

Reply:
Yes, I definitely imagined it as a constant, but made it a instance variable. :innocent: 

I have now changed it to a global constant, since `MIRROR_INCOMPATIBLE_REPO_ERR_MSG` which is somewhat related was also a global constant. It was that or `RpmFirstStage.UNSUPPORTED_METADATA`, since that would have scoped it to the only stage that actually uses it, but the same is true of `MIRROR_INCOMPATIBLE_REPO_ERR_MSG`, and I did not want to move that.